BadHabbit/BadHabbit/Kit/DesignKit/GlobalCoordinator/GlobalCoordinator.swift

```diff
@@ -15,7 +15,8 @@ protocol IGlobalCoordinator {
     )
     
     func presentOnTopVisibleViewController(_ viewController: UIViewController)
-
+    
+    func pushVC(_ viewController: UIViewController)
```

Comment:
In public functions don't use VC, full name only

Reply:
resolved